app/jobs/import_legislators_job.rb

```diff
@@ -0,0 +1,57 @@
+class ImportLegislatorsJob
+  include Worker
+
+  def initialize(redis = Redis.new, service = OpenStatesService.new)
+    @redis = redis
+    @service = service
+  end
+
+  def perform
+    import_date = @redis.get(:import_legislators_job_date)&.to_time
+
+    legislator_attrs = @service
+      .fetch_legislators(fields: fields, updated_since: import_date)
```

Comment:
did you find that the legislators search endpoint supports `updated_since`? based on their docs, its not one of the accepted parameters. if that's the case, this job won't need to write anything to redis, and it should only run on the first of the month (as a first pass at least). the `export_bills_job` does something like this so that it only runs on saturdays.

Reply:
There's a `updated_at` "The date that this object was last updated in our system" I was thinking about pointing it to. Not sure if that was the same as what we were looking for.

Reply:
updated_at is a field, but it doesn't look like theres a query parameter for date. the goal is to not fetch too much data. i think updating the job so that it doesn't check/store the redis date, and it instead does something like `return if today.is_first_of_month` as its first line will suffice. we should also kick off this job in the umbrella `import_job`

Reply:
yup https://github.com/legislated/legislated-api/pull/46/commits/6e38cf6eba1f3234cd661da99188af1c5e74467a

I'll change the job so it only fetches on the first of the month.